wordpress-mobile/WordPress-iOS
# Zendesk: Replace `ZendeskSDK` with `ZendeskSupportSDK`.

Fixes #13371 

This just updates Zendesk to use their new pod, `ZendeskSupportSDK`, since `ZendeskSDK` has been deprecated. So just smoke testing functionality should be sufficient.

To test:
- Go to Me > Help & Support.
- If you have not, set a 'Contact Email' (do not use your a8c email address).
- Create a ticket via any of these paths:
  - Contact Us
  - My Tickets
  - WordPess Help Center
- Go to My Tickets and verify your ticket appears there.
- Verify a ticket is created in Zendesk (it may take a few minutes).

PR submission checklist:
